charts/artifacts/values.yaml

```diff
@@ -49,11 +49,11 @@ cloudStorage:
 image:
   ## regitry to pull from.
   ##
-  registry: registry.scality.com
+  registry: ghcr.io
```

Comment:
Please confirm that all the tagged images have been migrated to ghcr.io/scality/artifacts` and that we can pull the image without any kind of authentication.

Reply:
Yes everything's good